flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -153,6 +153,30 @@
 		.defaultValue(false)
 		.withDescription("Monitor the current actual delayed write rate. 0 means no delay.");
 
+	public static final ConfigOption<Boolean> IS_WRITE_STOPPED = ConfigOptions
+		.key(RocksDBProperty.IsWriteStopped.getConfigKey())
+		.booleanType()
+		.defaultValue(false)
+		.withDescription("Monitor whether write has been stopped. 1 means that has been stopped.");
```

Comment:
`true` means that has been stopped.?

Reply:
This would return a numeric value, `1` means write has been stopped. I would explain this in detail to not mislead users.

Reply:
You could refer to above `MONITOR_ACTUAL_DELAYED_WRITE_RATE` , `0` means no delay. Similarity, `MONITOR_ACTUAL_DELAYED_WRITE_RATE`, `1` means write has been stopped.

Reply:
Suggest to follow the same description pattern as `CompactionPending`:
"Track whether write has been stopped in RocksDB. Returns 1 if write has been stopped, 0 otherwise."